src/display.rs

```diff
@@ -236,15 +236,16 @@ impl Display {
 
         // Initialize glyph cache
         let glyph_cache = {
-            info!("Initializing glyph cache");
+            trace!("initializing glyph cache");
             let init_start = ::std::time::Instant::now();
 
             let cache =
                 renderer.with_loader(|mut api| GlyphCache::new(rasterizer, &font, &mut api))?;
 
             let stop = init_start.elapsed();
-            let stop_f = stop.as_secs() as f64 + f64::from(stop.subsec_nanos()) / 1_000_000_000f64;
-            info!("Finished initializing glyph cache in {}", stop_f);
+            let stop_f = stop.as_secs() as f64 +
+                         f64::from(stop.subsec_nanos()) / 1_000_000_000f64;
+            trace!("finished initializing glyph cache in {}s", stop_f);
```

Comment:
I'd like to keep this at info level. It's not like we've got too much stuff at info currently. I'd be fine with knocking it down to debug though.